### DISCUSSION TITLE
Wanted: maintainers & reviewers

### DISCUSSION THREAD

**Discussion by plaimbock** [tabler/tabler#1996] [General]
Hi,

Recently I asked Paweł if he needed any help with Tabler. His response:

"Yes, I'm open for any collaboration which can help to improve Tabler.
Can you give me your github account? If you have more collaborators which can help me, just let me know about it. 
I'm a little busy, I want to improve Tabler, but at this moment I dont have a lot of time to do it. "

So if you are interested in helping out as a maintainer or reviewer please let me know (include your gh account) so I can forward a list to Paweł for his consideration.

Also feel free to add your comments to any PR. Something as simple as LGTM (Looks Good To Me) also helps moving things forward.

Best,
Patrick

**Comment by georgedobre** [comment]
Hi, i'm interested in contributing to the project with code and $$$ if needed

**Reply by plaimbock** [reply]
Hi George,

Great! I have forwarded your name to Paweł for consideration.

Best,
Patrick

**Comment by abinoam** [comment]
I'm starting to use it in one simple Rails project. I'd be glad to help. At first, I'll try to document any difficult, any patch any thing that I did in this project, and try to bring it to the issues, PRs and discussions of the repository so it may help others.

**Comment by pkaramagi** [comment]
i am interested as well
